sr_health_check/src/sr_health_check/sr_health_check_gui.py

```diff
@@ -159,6 +207,27 @@ def button_start_all_clicked(self):
                 threading.Thread(target=self._checks_to_execute[check_name]['check'].run_check)
             self._check_queue.put(self._checks_to_execute[check_name])
 
+    '''
+        Performs actions upon 'Stop' button click.
+    '''
+    def _button_stop_clicked(self):
+
+        for check_name in self._check_names:
+            self._checks_to_execute[check_name]['check'].stop_test()
+
+        while not self._check_queue.empty() and not rospy.is_shutdown():
```

Comment:
I've seen the same check **not self._check_queue.empty()** in line 132. Perhaps it would be nice to have a method that checks if the checks are running to make this more readable.

Reply:
so basically this empty() method tells me if there are any checks pending to be executed, but I'll wrap it into are_checks_running() method for better readability, is that fine with you?